Dockerfile

```diff
@@ -44,6 +44,8 @@ RUN set -ex; \
     '; \
     apt-get update; \
     apt-get install -y $buildDeps --no-install-recommends; \
+    # Since there's no confluent-kafka wheel for aarch64, remove when there is
+    [ $(uname -m) = "aarch64" ] && apt-get install -y librdkafka-dev --no-install-recommends; \
```

Comment:
For the Intel build, this library is not necessary since there's an Intel wheel.
Hopefully one day `confluent-kafka` will have a wheel and this will not be required.
There's also a chance I might be able to produce it but for now let's use this.